plugins/t/check_dns.t

```diff
@@ -58,7 +58,7 @@ my $dns_server       = getTestParameter(
 my $host_nonresponsive = getTestParameter(
 			"NP_HOST_NONRESPONSIVE",
 			"The hostname of system not responsive to network requests",
-			"10.0.0.1",
+			"192.0.2.0",
```

Comment:
is this change really intended here?

Reply:
No. I can't recall why I changed that...